fairmq/nanomsg/FairMQTransportFactoryNN.cxx

```diff
@@ -63,7 +63,8 @@ FairMQPollerPtr FairMQTransportFactoryNN::CreatePoller(const FairMQSocket& cmdSo
 
 void FairMQTransportFactoryNN::Shutdown()
 {
-    nn_term();
+    // nn_term();
+    // see https://www.freelists.org/post/nanomsg/Getting-rid-of-nn-init-and-nn-term,8
 }
```

Comment:
Does it cause any issues? I would prefer to leave `nn_term()` in. Even though timeouts on socket transfer operations should make sure none of our sockets hang when the state machine wants to terminate, better to be on the safe side. It is same semantic as `zmq_ctx_shutdown()`. The post by Sustrik is 3 years old, things might have changed or change in the future. They still have it documented and used in their tests.